orchestration/dagster/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "dagster_poc"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+dagster = "^0.10.5"
+dagit = "^0.10.5"
+google-cloud-storage = "^1.3.5"
+dagster-k8s = "^0.10.5"
+data-repo-client = "^1.0.194.post5"
+
+[tool.poetry.dev-dependencies]
+pdbpp = "^0.10.2"
+pytest = "^6.2.1"
+
```

Comment:
can add a [tool.poetry.scripts] section to have certain preconfigured scripts, idk if that'll help us out so much for this. something to keep an eye on moving forward

Reply:
@raaidbroad not sure I follow, can you elaborate? What preconfigured scripts are you referring to?

Reply:
https://python-poetry.org/docs/pyproject/#scripts so we can point to specific functions in specific modules, if that would be desired. Nice clean way to run stuff depending on the use case.